ddtrace/contrib/pytest/_efd_utils.py

```diff
@@ -254,13 +332,29 @@ def efd_pytest_terminal_summary_post_yield(terminalreporter: _pytest.terminal.Te
     # Print summary counts
     terminalreporter.write_sep("_", "Datadog Early Flake Detection summary", purple=True, bold=True)
 
-    terminalreporter.write_sep(
-        " ",
-        markedup_summary_string,
-        fullwidth=terminalreporter._tw.fullwidth + (len(markedup_summary_string) - len(raw_summary_string)),
-        purple=True,
-        bold=True,
-    )
+    if raw_summary_string:
+        terminalreporter.write_sep(
+            " ",
+            markedup_summary_string,
+            fullwidth=terminalreporter._tw.fullwidth + (len(markedup_summary_string) - len(raw_summary_string)),
+            purple=True,
+            bold=True,
+        )
+    else:
+        if InternalTestSession.efd_is_faulty_session():
+            terminalreporter.write_sep(
+                " ",
+                "Too many new tests detected (session considered faulty).",
```

Comment:
I think it might not be clear to a user what 'faulty' means here. It means we probably failed to calculate new tests correctly, right?